### DISCUSSION TITLE
AZT2316A datasheet

### DISCUSSION THREAD

**Discussion by headsetguy37** [86Box/86Box#1744] [General]
Does anyone have a datasheet for the Aztech AZT2316A chipset? I want to see what some of the differences are between that and the OPTi 82C929A.

**Comment by Ompronce** [comment]
> Does anyone have a datasheet for the Aztech AZT2316A chipset? I want to see what some of the differences are between that and the OPTi 82C929A.

I have also been looking for a datasheet for the AZT1605 and AZT2316 sound cards, and I haven't been able to find them either. Regarding the OPTi datasheets I was able to find the datasheets for the 82C929 and the 82C930 sound cards (the main difference is that the 82C930 has the WSS codec integrated instead of separate).
